Add missing VFP-v3/Neon D0-D31 registers (#4589)

ahmed-hany94 opened the pull request:
# SQUASH ME

[Per issue #4589: DWARF: Missing some ARM32 registers](https://github.com/rizinorg/rizin/issues/4589) VFP-v3/Neon D0-D31 registers were missing from `map_dwarf_reg_to_arm32` causing

```
WARNING: (libr/arch/isa/arm/arm_dwarf_regnumb_table.h:137):
map_dwarf_reg_to_arm32: code should not be reached
```

 <!-- Filling this template is mandatory -->

**Your checklist for this pull request**
- [ ] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [ ] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [ ] I've documented or updated the documentation of every function and struct this PR changes. If not so I've explained why.
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the [rizin book](https://github.com/rizinorg/book) with the relevant information (if needed)

**Detailed description**

<!-- Explain the **details** for making this change. Is a new feature implemented? What existing problem does the pull request solve? How does the pull request solve these issues? Please provide enough information so that others can review your pull request. -->

...

**Test plan**

<!-- What steps should the reviewer take to test your pull request? Demonstrate the code is solid. Example: The exact commands you ran and their output, screenshots/videos. This is your time to re-check that everything works and that you covered all the edge cases -->

...

**Closing issues**

<!-- put "closes #XXXX" in your comment to auto-close the issue that your PR fixes (if any). -->

...

wargio reviewed (commented):
what a weird diff.

wargio reviewed (changes requested):
Found the issue: **Please push the changes using UNIX line delimiters. not windows ones.**

wargio reviewed (approved):
I have fixed the line ending, squash is mandatory

ahmed-hany94 committed:
Add missing VFP-v3/Neon D0-D31 registers

Per [issue: DWARF: Missing some ARM32
registers](https://github.com/rizinorg/rizin/issues/4589)
VFP-v3/Neon D0-D31 registers were missing from map_dwarf_reg_to_arm32
causing

```
WARNING: (libr/arch/isa/arm/arm_dwarf_regnumb_table.h:137):
map_dwarf_reg_to_arm32: code should not be reached
```
